### PR TITLE
Move into Shared for SqlInfoMessageEvent.cs

### PR DESCRIPTION
**Pull request by lcheunglci**
Relates to #1261 .  Move the netcore version of SqlInfoMessageEvent.cs to shared src and updated the references for the csprojs.

### CODE REVIEW

**Comment by Kaur-Parminder** [comment]
@lcheunglci There are couple of MDAC codes in netfx. Please move those as well to common file.

File: src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlInfoMessageEvent.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        internal SqlInfoMessageEventArgs(SqlException exception)
+        {
+            _exception = exception;
+        }
+
+        /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlInfoMessageEventArgs.xml' path='docs/members[@name="SqlInfoMessageEventArgs"]/Errors/*' />
+        public SqlErrorCollection Errors
+        {
+            get { return _exception.Errors; }
+        }
```
Comment (johnnypham) [inline src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlInfoMessageEvent.cs:21]:
```suggestion
        public SqlErrorCollection Errors => _exception.Errors;
```

File: src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlInfoMessageEvent.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+        private bool ShouldSerializeErrors()
+        { // MDAC 65548
+            return (null != _exception) && (0 < _exception.Errors.Count);
+        }
```
Comment (johnnypham) [inline src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlInfoMessageEvent.cs:26]:
```suggestion
        // MDAC 65548
        private bool ShouldSerializeErrors() => (null != _exception) && (0 < _exception.Errors.Count);
```

File: src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlInfoMessageEvent.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+        /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlInfoMessageEventArgs.xml' path='docs/members[@name="SqlInfoMessageEventArgs"]/Message/*' />
+        public string Message
+        { // MDAC 68482
+            get { return _exception.Message; }
+        }
```
Comment (johnnypham) [inline src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlInfoMessageEvent.cs:32]:
```suggestion
        public string Message => _exception.Message;
```

File: src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlInfoMessageEvent.cs [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+        /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlInfoMessageEventArgs.xml' path='docs/members[@name="SqlInfoMessageEventArgs"]/Source/*' />
+        public string Source
+        { // MDAC 68482
+            get { return _exception.Source; }
+        }
```
Comment (johnnypham) [inline src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlInfoMessageEvent.cs:38]:
```suggestion
        // MDAC 68482
        public string Source => _exception.Source;
```

File: src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlInfoMessageEvent.cs [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+        /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlInfoMessageEventArgs.xml' path='docs/members[@name="SqlInfoMessageEventArgs"]/ToString/*' />
+        override public string ToString()
+        { // MDAC 68482
+            return Message;
+        }
```
Comment (johnnypham) [inline src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlInfoMessageEvent.cs:44]:
```suggestion
        // MDAC 68482
        override public string ToString() => Message;
```

**Comment by lcheunglci** [comment]
Thanks @johnnypham for the expression bodied getters suggestion.

**Review by cheenamalhotra — APPROVED** [review]

**Pull request merged**